packages/excalidraw/components/App.tsx

```diff
@@ -4146,10 +4146,24 @@ class App extends React.Component<AppProps, AppState> {
           | ExcalidrawArrowElement
           | undefined;
 
-        const step = elbowArrow
-          ? elbowArrow.startBinding || elbowArrow.endBinding
-            ? 0
-            : ELEMENT_TRANSLATE_AMOUNT
+        const preventKeyboardMove = selectedElements
+          .filter(isElbowArrow)
+          .find((arrow) => {
+            const startMissing =
+              arrow.startBinding &&
+              !selectedElements.find(
+                (el) => el.id === arrow.startBinding?.elementId,
+              );
+            const endMissing =
+              arrow.endBinding &&
```

Comment:
Why check for this if in the `find()` predicate `el.id` will never equal `undefined` if the `endBinding` itself is `undefined`.

Reply:
and in that case find will return undefined, however for clarity I changed it to some.

Reply:
ok. I am super slow today... I removed arrow.endBinding &&

Reply:
Thanks. I'd wait for @dwelle whether he agrees with my feedback. I can see how pre-checking the existence of the binding can save on cycle count, but I'm on the fence whether it's worth it.

Reply:
I told you I am slow today. `arrow.endBinding &&` is there because the element is only missing if there should be something bound, but it is not there.